external/st_audiofile/src/st_audiofile.c

```diff
@@ -566,15 +567,16 @@ uint64_t st_read_f32(st_audio_file* af, float* buffer, uint64_t count)
             if (!buf_i32) {
                 return 0;
             }
-            count = channels * WavpackUnpackSamples(af->wv, buf_i32, (uint32_t)count);
-            if (!(af->cache.wv.mode & MODE_FLOAT)) {
+            count = WavpackUnpackSamples(af->wv, buf_i32, (uint32_t)count);
+            {
```

Comment:
I guess from the code above the if condition was always true. If so, we can remove the brackets too.

Reply:
Yes, we can.